UEBPrep/spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
We really need basic testing for the user model and controller, as this is one of the core functionality of our apps we need to have high assurance it is working as we intend.